.vscode/cspell.json

```diff
@@ -159,6 +160,7 @@
     "skus",
     "Somaxconn",
     "southcentralus",
+    "spatio",
```

Comment:
Please move this into an area specific to your service.  This is not a general concept that we want other libraries using.